docs/src/design/decoder/constraints.md

```diff
@@ -38,6 +39,12 @@ When `SPLIT` or `LOOP` operation is executed, the top of the operand stack must
 (f_{split} + f_{loop}) \cdot (s_0^2 - s_0) = 0 \text{ | degree} = 7
 $$
 
+When a `DYN` operation is executed, the hasher registers must all be set to $0$:
+
+> $$
+f_{dyn} \cdot (1 - h_i) = 0 \text { for } i \in [0, 8) \text{ | degree} = 2
+$$
```

Comment:
Should this not be degree $6$ since degree of $f_{dyn}$ is $5$?

Reply:
Yes! That's definitely a typo. I'll fix it in a follow-up PR